qemu/tests/cfg/passthrough_with_multipath_device.cfg

```diff
@@ -0,0 +1,24 @@
+- passthrough_with_multipath_device:
+    virt_test_type = qemu
+    type = passthrough_with_multipath_device
+    kill_vm = yes
+    login_timeout = 240
+    start_vm = no
+    only Linux
+    pre_command = "yum install device-mapper-multipath -y;mpathconf --enable;systemctl restart multipathd;"
+    pre_command += "modprobe scsi_debug dev_size_mb=5000 num_tgts=1 vpd_use_hostno=0 add_host=2 delay=1 max_luns=2 no_lun_0=1;sleep 5"
+    post_command = "multipath -F;sleep 3;modprobe -r scsi_debug"
+    cmd_dd = "for((;;)) do dd if=/dev/zero of=%s bs=1M count=5000 oflag=direct; done"
+    images += " stg0"
+    image_format_stg0 = raw
+    drive_format_stg0 = scsi-block
+    image_raw_device_stg0 = yes
+    offline_time = 10
+    repeat_time = 10
+    get_id_cmd = "multipath -l | grep scsi_debug | tr -d "()" | awk 'NR==1{print $2}'"
+    get_mpath_cmd = "multipath -l %s | grep %s | awk 'NR==1{print $1}'"
+    get_mdev_cmd = "lsscsi -ik | grep %s | awk '{print $6}'"
+    get_tdev_cmd = "lsscsi|grep scsi_debug | awk '{print $6}'"
```

Comment:
There is underlying issue is the device not created by scsi_debug, so far it is acceptable.